service/version_bundles.go

```diff
@@ -30,6 +30,16 @@ func newVersionBundles() []versionbundle.Bundle {
 					Description: "Updated to version 3.0.1.",
 					Kind:        versionbundle.KindChanged,
 				},
+				{
```

Comment:
no comments, only thought in my mind: "data" and "code" should live SEPARATELY.

Reply:
I agree with you in general, but TBH there is no code. This is pure data file if you analize it. We could name it `version_bundles.json`, read and parse to VersionBundle struct, but there is no reason to do so. You can think about it as "Go Object Notation" format, like "JavaScript Object Notation" format, aka. JSON.